packages/swingset-liveslots/src/collectionManager.js

```diff
@@ -942,7 +942,12 @@ export function makeCollectionManager(
     return collectionToWeakSetStore(reanimateCollection(vobjID));
   }
 
-  const testHooks = { obtainStoreKindID, storeSizeInternal, makeCollection };
+  const testHooks = {
+    obtainStoreKindID,
+    storeSizeInternal,
+    makeCollection,
+    allCollectionObjIDs,
```

Comment:
This will conflict with the removal of `allCollectionObjIDs` in PR #7333 (currently 2nd in your review queue, on top of PR #7354). Which ever one lands last will need some slight fixes (I bet git will think these can merge cleanly, when in fact there will be semantic breakage, because this symbol is deleted in a non-conflicting chunk).

Given the upcoming removal, I'd suggest not including it in testHooks or getRetentionStats.

That PR introduces a couple of new caches, but since they're all flushed at the end of each crank, their size at `getRetentionStats()` time (end of BOYD) is always going to be zero, so it wouldn't be useful to include them in `getRetentionStats` either. So maybe omit `getRetentionStats` entirely, or retain it as a stub but have it return an empty record.